tests/ostree_test.cc

```diff
@@ -27,10 +29,11 @@ TEST(OstreePackage, ToEcuVersion) {
 
 TEST(OstreePackage, InstallBadUri) {
   OstreePackage op("branch-name-hash", "hash", "bad_uri");
-  data::PackageManagerCredentials cred;
   Config config;
   config.pacman.type = kOstree;
   config.pacman.sysroot = sysroot;
+  CryptoKey keys(boost::make_shared<FSStorage>(FSStorage(config.storage)), config);
```

Comment:
I get an error when I test locally here because I don't have write access to `/var/sota`. We need to set `config.storage.metadata_path` to something temporary in the build directory.

Reply:
Updated, waiting for CI finish